pinot-core/src/main/java/com/linkedin/pinot/core/query/scheduler/QueryScheduler.java

```diff
@@ -122,6 +122,8 @@ public void stop() {
     try {
       dataTable = queryExecutor.processQuery(queryRequest, executorService);
     } catch (Exception e) {
+      LOGGER.error("Encountered exception while processing requestId {} from broker {}",
```

Comment:
anyway we can rate limit this?

Reply:
I did consider that, and arrived at this change for the following reasons:
- We have OmitStackTraceInFastThrow enabled, so hopefully that will control some of the damage.
- Broker currently logs the error for this (except that I couldn't find a good way to include server context there) and I dont' see it being a very high rate. This still doesn't mean we won't flood the logs IF something bad tickles the server, but I am guessing the chances of that are low.
- The linked jira has context on the specific error and I could log the error in processQuery for the specific case we are chasing. It required converting a precondition check into a log and throw, which seemed clunky to me and having the caller take care of logging exceptions seemed like a better approach.

Let me know if you think its better to track counts per interval and only log them once in a while. I am however not sure how useful that would be in debugging issues.

Reply:
Will merge for now and update with rate limiting as necessary.